test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecOperation.java

```diff
@@ -206,6 +206,7 @@ private AccountID randomNodeFrom(HapiApiSpec spec) {
 				updateStateOf(spec);
 			}
 		} catch (Throwable t) {
+			t.printStackTrace();
```

Comment:
As noted on #1751, this blows up our regression suite. 😝

Reply:
Seems like merge leftover from refactored nfts touchups. I've removed, so regression suite can commence